src/lib/chips/chips.scss

```diff
@@ -0,0 +1,18 @@
+$md-chip-vertical-padding: 8px;
+$md-chip-horizontal-padding: 12px;
+
+.md-chip-list {
+  padding: $md-chip-horizontal-padding;
+}
+
+.md-chip {
+  display: inline-block;
+  padding: $md-chip-vertical-padding $md-chip-horizontal-padding
+    $md-chip-vertical-padding $md-chip-horizontal-padding;
+  border-radius: $md-chip-horizontal-padding * 2;
+
+  background-color: #E0E0E0;
+  color: rgba(0, 0, 0, 0.87);
```

Comment:
All colors should be in a `_chips-theme.scss` file. See the other components for an example.